### PR TITLE
Add node.js Environment

### PR DESCRIPTION
**Pull request by BSU-EzekielHolmes**
I've added some code to our workflow file which will attempt to run test files that don't exist yet (that'll be what I work on next). This updated code adds a node.js environment to the ubuntu instance the workflow creates so that the test files can be run.

### CODE REVIEW

**Comment by BSU-EzekielHolmes** [comment]
I don't really know what I'm doing so this could be wrong but I believe what I have going right now is there is a node.js environment setup and I also believe at this point the lint (14.x), lint(16,x) and lint(19.x) jobs should fail since the environment is attempting to run tests that don't exist yet. I can go ahead and write a couple tests before we approve this PR jus to see if it works.

**Comment by BSU-EzekielHolmes** [comment]
#141

**Comment by BSU-EzekielHolmes** [comment]
package-lock.json and package.json will need to be modified and for implementing unit testing, I believe a new file containing those tests needs to be created. Right now, none of the Node.js CI workflow jobs are actually doing much, just simply sets up an environment for the tests to run in once we have created them.

**Review by mariotorres1 — APPROVED** [review]

**Review by mitchcrocker — APPROVED** [review]

**Pull request merged**
